open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/service-worker.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
This again doesn't look like our source? Does it need to be checked in - I'll skip further references to this, but would be good to clarify, as if we are reshipping we need to figure how to meet terms of license

Reply:
License changed.
It is generated by the polymer starter kit.

Reply:
as above (ok to close this issue as above one tracks it)